src/utils/local-storage-versioning.ts

```diff
@@ -0,0 +1,10 @@
+import { APP_VERSION } from './constants';
+
+export const checkLocalStorageVersion = () => {
```

Comment:
naming. skal vi bruke begge begrepene. LocalStorageVersion og app-version ?

Reply:
det er to forskjellige konstanter. APP_VERSION er den vi har lokalt (som vi må oppdatere), mens localStorageVersion er den versjonen som er lagret i localStorage